src/ui/scss/component/_main.scss

```diff
@@ -35,7 +49,7 @@
     'content content'
     'info    related';
 
-  @media (min-width: 1470px) {
+  @media (min-width: 1300px) {
```

Comment:
should this be a constant? is this value used elsewhere?

Reply:
Maybe. We aren't using it anywhere else. I just wanted to lower it so I can see the related content to the right on my laptop.